testdata/jennies/builders/package-with-dashes/TypescriptBuilder/src/builder-pkg/somenicebuilder_builder_gen.ts

```diff
@@ -0,0 +1,19 @@
+import * as cog from '../cog';
+import * as with-dashes from '../with-dashes';
```

Comment:
I think this should be replaced too. I think the same logic as go would look better than a dash here. Or even better would be to capitalize
```suggestion
import * as withDashes from '../with-dashes';
```